frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -346,21 +361,30 @@ export const TimespanDropdown = () => {
       items={timespanOptions}
       label={t('public~Time range')}
       onChange={onChange}
-      selectedKey={endTime ? CUSTOM_TIME_RANGE_KEY : formatPrometheusDuration(timespan)}
+      selectedKey={
+        endTime || endTimeFromParams
+          ? CUSTOM_TIME_RANGE_KEY
+          : formatPrometheusDuration(_.toNumber(timeSpanFromParams) || timespan)
+      }
     />
   );
 };
 
 export const PollIntervalDropdown = () => {
   const { t } = useTranslation();
-
+  const refreshIntervalFromParams = getQueryArgument('refreshInterval');
   const interval = useSelector(({ UI }: RootState) =>
     UI.getIn(['monitoringDashboards', 'pollInterval']),
   );
 
   const dispatch = useDispatch();
   const setInterval = React.useCallback(
-    (v: number) => dispatch(monitoringDashboardsSetPollInterval(v)),
+    (v: number) => {
+      v
```

Comment:
I think we should avoid using the ternary operator for control flow. In other words, prefer `if (v) { ... } else { ... }` in this case.